test/system/irohad_test.cpp

```diff
@@ -154,9 +170,114 @@ class IrohadTest : public AcceptanceFixture {
+        shared_model::interface::permissions::Role::kGetMyAccAst,
+        shared_model::interface::permissions::Role::kGetMyAccAstTxs,
+        shared_model::interface::permissions::Role::kGetMyAccDetail,
+        shared_model::interface::permissions::Role::kGetMyAccTxs,
+        shared_model::interface::permissions::Role::kGetMyAccount,
+        shared_model::interface::permissions::Role::kGetMySignatories,
+        shared_model::interface::permissions::Role::kGetMyTxs,
+        shared_model::interface::permissions::Role::kReceive,
+        shared_model::interface::permissions::Role::kRemoveSignatory,
+        shared_model::interface::permissions::Role::kSetQuorum,
+        shared_model::interface::permissions::Role::kTransfer};
+
+    shared_model::interface::RolePermissionSet money_perms{
+        shared_model::interface::permissions::Role::kAddAssetQty,
+        shared_model::interface::permissions::Role::kCreateAsset,
+        shared_model::interface::permissions::Role::kReceive,
+        shared_model::interface::permissions::Role::kTransfer};
+
+    auto genesis_tx_builder =
+        shared_model::proto::TransactionBuilder()
+            .creatorAccountId(kAdminId)
+            .createdTime(iroha::time::now())
+            .addPeer("0.0.0.0:10001", node0_keys.get().publicKey())
+            .createRole("admin", admin_perms)
+            .createRole(kDefaultRole, default_perms)
+            .createRole("money_creator", money_perms)
+            .createDomain(kDomain, kDefaultRole)
+            .createAsset(kAssetName, kDomain, 2)
+            .createAccount(kAdminName, kDomain, admin_keys.get().publicKey())
+            .createAccount("user", kDomain, user_keys.get().publicKey())
+            .appendRole(kAdminId, "admin")
+            .appendRole(kAdminId, "money_creator")
+            .quorum(1);
+
+    auto genesis_tx = genesis_tx_builder.build()
+                          .signAndAddSignature(node0_keys.get())
+                          .finish();
+
+    auto genesis_block =
+        shared_model::proto::BlockBuilder()
+            .transactions(
+                std::vector<shared_model::proto::Transaction>{genesis_tx})
+            .height(1)
+            .prevHash(shared_model::crypto::DefaultHashProvider::makeHash(
+                shared_model::crypto::Blob("")))
+            .createdTime(iroha::time::now())
+            .build()
+            .signAndAddSignature(node0_keys.get())
+            .finish();
+
+    std::ofstream output_file(path_genesis_.string());
+
+    shared_model::proto::ProtoBlockJsonConverter()
+        .serialize(genesis_block)
+        .match([&output_file](
+                   auto &&json) { output_file << std::move(json.value); },
+               [](const auto &error) {
+                 // should not get here
+                 std::exit(EXIT_FAILURE);
+               });
+  }
+
+  void removeTestData() {
+    boost::filesystem::remove(test_data_path_ / "node0.priv");
```

Comment:
Manual listing of each file does not look like a reliable approach.
Can we remove the whole `test_data_path_` instead?

Reply:
If so, then we have to move `config.sample` out from `test_data_path_`.